stdlib/os/__init__.pyi

```diff
@@ -624,11 +623,12 @@ if sys.platform != "win32":
 
     def pread(__fd: int, __length: int, __offset: int) -> bytes: ...
     def pwrite(__fd: int, __buffer: bytes, __offset: int) -> int: ...
+    # TODO: stubtest says preadv and pwritev are only available on macOS on 3.11+ -- is stubtest right?
```

Comment:
I have preadv and pwritev on my laptop on 3.7 and 3.10 as well

Reply:
Weird that stubtest has denied their existence up till now!

Reply:
```suggestion
    # In CI, stubtest sometimes reports that these are available on MacOS, sometimes not
```